Formula/grap.rb

```diff
@@ -1,9 +1,8 @@
 class Grap < Formula
   desc "Language for typesetting graphs"
   homepage "https://www.lunabase.org/~faber/Vault/software/grap/"
-  url "https://www.lunabase.org/~faber/Vault/software/grap/grap-1.45.tar.gz"
-  mirror "https://deb.debian.org/debian/pool/main/g/grap/grap_1.45.orig.tar.gz"
```

Comment:
missing the mirror one?

Reply:
Debian doesn't have this update yet. I believe the mirror was added due to the canonical URL being unreliable due to certificate issues in the past, but it looks like that's been fixed.

Reply:
I see, sounds like another place that we can have our own brew mirror.